### PR TITLE
Add man page for hexagonrpcd

### PR DESCRIPTION
**Pull request by xelageo**
This a basic man page for hexagonrpcd created for the future debian package. I've added the meson install_man.

### CODE REVIEW

File: hexagonrpcd/hexagonrpcd.1 [hunk 1]
```diff
@@ -0,0 +1,54 @@
+'\" t
+.TH HEXAGONRPCD 1 "2024-09-28" "hexagonrpcd" "Server for FastRPC remote procedure calls from Qualcomm DSPs"
+.SH NAME
+hexagonrpcd - Server for FastRPC remote procedure calls from Qualcomm DSPs
+.SH SYNOPSIS
+\fBhexagonrpcd\fP [\fIparameters\&.\&.\&.\fP]
```
Comment (flamingradian) [inline hexagonrpcd/hexagonrpcd.1:6]: The `-f DEVICE` option is required as shown in the help text, but this omits that information.

File: hexagonrpcd/hexagonrpcd.1 [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+.PP
+.SH PARAMETERS
+.TP
+\fB\-d DSP\fP
+DSP name (default: )
+.TP
+\fB\-f DEV\fP
+FastRPC device node to attach to
+.TP
+\fB\-p program\fP
```
Comment (flamingradian) [inline hexagonrpcd/hexagonrpcd.1:18]: All-lowercase "program" suggests that the user must literally put `hexagonrpcd -p program`, which is not the case. The intended use case is something like `hexagonrpcd -p chrecd` or `hexagonrpcd -p sh`.

File: hexagonrpcd/hexagonrpcd.1 [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+.PP
+.SH NOTES
+
+Hexagonrpcd serves files to the remote processor via a reverse tunnel interface
+to the remote processor. These files are by default searched for in:
+.TS
+l l
+---
+l ll. 
+Physical file/dir       Android file/dir
```
Comment (flamingradian) [inline hexagonrpcd/hexagonrpcd.1:35]: Is there some specific reason to align the table like this?

File: hexagonrpcd/hexagonrpcd.1 [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+Root directory of served files (default: /usr/share/qcom/)
+.TP
+\fB\-s\fP
+Attach to sensorspd 
+.PP
+.SH NOTES
+
+Hexagonrpcd serves files to the remote processor via a reverse tunnel interface
+to the remote processor. These files are by default searched for in:
+.TS
```
Comment (flamingradian) [inline hexagonrpcd/hexagonrpcd.1:31]: Empty line before, please. I'm testing this and the table looks connected to the paragraph.

File: hexagonrpcd/hexagonrpcd.1 (hunk 1 above)
Comment (ebolalex) [reply hexagonrpcd/hexagonrpcd.1]: Should be corrected, totally missed that

File: hexagonrpcd/hexagonrpcd.1 (hunk 3 above)
Comment (ebolalex) [reply hexagonrpcd/hexagonrpcd.1]: No, that was the result of my experiment without the `'\" t` at the start of the man page

File: hexagonrpcd/hexagonrpcd.1 (hunk 4 above)
Comment (ebolalex) [reply hexagonrpcd/hexagonrpcd.1:32]: Done

File: hexagonrpcd/hexagonrpcd.1 (hunk 2 above)
Comment (xelageo) [reply hexagonrpcd/hexagonrpcd.1]: I changed the case in the new man version

**Comment by flamingradian** [comment]
Removed trailing whitespace, squashed, and merged.

**Pull request merged**